packages/relay/tests/lib/eth/eth-config.ts

```diff
@@ -572,6 +572,9 @@ export const DEFAULT_LOGS = {
 export const DEFAULT_ETH_GET_BLOCK_BY_LOGS = {
   logs: [DEFAULT_LOGS.logs[0], DEFAULT_LOGS.logs[1]],
 };
+
+export const DEFAULT_ROOT_HASH = '0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421';
```

Comment:
Q: Where is this root hash from? Is there a document referencing this root hash? If so, please add a comment with the link to the document above it. The hash seems a bit arbitrary and random to me.

Reply:
Added a short description